extensions/omni_ext/tests/tests.yml

```diff
@@ -0,0 +1,130 @@
+instances:
+  with_omni_ext:
+    default: true
+    config:
+      shared_preload_libraries: */env/OMNI_EXT_SO
+    init:
+    - create extension omni_ext cascade
+
+  with_omni_ext_test:
+    config:
+      shared_preload_libraries: */env/OMNI_EXT_SO
+    init:
+      - create extension omni_ext_test cascade
+
+  with_omni_ext_test_no_preload:
+    config:
+      shared_preload_libraries: */env/OMNI_EXT_SO
+    init:
+      - create extension omni_ext_test_no_preload cascade
+
+tests:
+  - name: load
+    query: select omni_ext.load('omni_ext_test')
+    results:
+      - load: 0.1
+
+  - name: load version
+    steps:
+      - query: select omni_ext.load('omni_ext_test', 'unknown')
+        results:
+          - load:
+      - query: select omni_ext.load('omni_ext_test', '0.1')
+        results:
+          - load: 0.1
+
+  - name: shmem
+    instance: with_omni_ext_test
+    steps:
+      - name: Should be initialized since it's preloaded
+        query: select omni_ext_test.alloc_shmem_global()
+        results:
+          - alloc_shmem_global: test
+
+      - name: database-local
+        query: select omni_ext_test.alloc_shmem_database_local()
+        results:
+          - alloc_shmem_database_local: 'testdb 0'
+      
+#CREATE DATABASE another_db;
+#\connect another_db
+#CREATE EXTENSION omni_ext_test CASCADE;
+#NOTICE:  installing required extension "omni_ext"
+#SELECT omni_ext_test.alloc_shmem_database_local();
+# alloc_shmem_database_local 
+#----------------------------
+# testdb 1
+#(1 row)
+
+  - name: reloading
+    instance: with_omni_ext_test
+    steps:
+      - name: preloaded value
+        query: select omni_ext_test.alloc_shmem_global()
+        results:
+        - alloc_shmem_global: test
+      - name: modify value
+        query: select omni_ext_test.update_global_value('updated')
+        results:
+          - update_global_value: updated
+      - name: re-load again
+        query: select omni_ext.load('omni_ext_test')
+        results:
+          - load: 0.1
+      - name: should not be rolled back to the initial value
+        query: select omni_ext_test.alloc_shmem_global()
+        results:
+        - alloc_shmem_global: updated
+        
+
+  - name: database local bgworker
+    instance: with_omni_ext_test
+    steps:
+    - query: select omni_ext_test.wait_for_table('local_worker_started')
+      results:
+      - wait_for_table: true
+    
+    #\d global_worker_started;
```

Comment:
is this (and below) a placeholder for something or just a comment?

Reply:
Right - wanted to get your thoughts on it too. We are describe the table, but not expecting a value. We can re-implement `\d` in SQL - but it could take some effort, so I wanted to know if it was a really necessary test then I can go ahead and do that.

Reply:
> We can re-implement \d in SQL - but it could take some effort

Just in case it helps. You can do `\set ECHO_HIDDEN` in `psql` to get the queries executed by any slash command (like `\d`).